docs/en/observability/monitor-nginx.asciidoc

```diff
@@ -0,0 +1,249 @@
+[[monitor-nginx]]
+= Monitor Nginx: Observe the logs and metrics of your Nginx instances
```

Comment:
Looking at the [NGINX website and documentation](https://docs.nginx.com/), I believe the official product name is `NGINX` (all caps). I think we should update all instances of NGINX in these docs to match that.

Reply:
Yeah, we had a discussion about this in slack. Ideally, the integration would be named correctly. I originally had NGINX throughout, except when calling out the integration, but that feels a little weird to me as well. "nginx" all-lowercase is also a copyrighted term, and nginx.org uses all lowercase. Maybe all lowercase sticks out a little less than all caps when compared to our integration name.

Reply:
👍 sgtm